docs/src/pages/basics/guide-react/index.md

```diff
@@ -5,41 +5,41 @@ title: 'Storybook for React'
 
 You may have tried to use our quick start guide to setup your project for Storybook. If you want to set up Storybook manually, this is the guide for you.
 
-> This will also help you to understand how Storybook works.
+This will also help you to understand how Storybook works.
 
 ## Starter Guide React
 
-Storybook has its own Webpack setup and a dev server. Webpack setup is very similar to [Create React App](https://github.com/facebookincubator/create-react-app), but allows you to configure as you want.
+Storybook has its own Webpack setup and a dev server. Webpack setup is very similar to [Create React App](https://github.com/facebookincubator/create-react-app), but allows you to configure it as you want.
 
 In this guide, we will set up Storybook for your React project.
 
 ## Table of contents
 
 -   [Add @storybook/react](#add-storybookreact)
--   [Add react, react-dom, and babel-core](#add-react-react-dom-and-babel-core)
+-   [Add react, react-dom, babel-core, and babel-loader](#add-react-react-dom-babel-core-and-babel-loader)
 -   [Create the config file](#create-the-config-file)
 -   [Write your stories](#write-your-stories)
 -   [Run your Storybook](#run-your-storybook)
 
 ## Add @storybook/react
 
-First of all, you need to add `@storybook/react` to your project. To do that, simply run:
+First of all, you need to add `@storybook/react` to your project. To do that, run:
 
 ```sh
 npm i --save-dev @storybook/react
 ```
 
 ## Add react, react-dom, babel-core, and babel-loader
 
-Make sure that you have `react`, `react-dom`, `@babel/core`, and `babel-loader` in your dependencies as well because we list these as a peerDependency:
+Make sure that you have `react`, `react-dom`, `babel-core`, and `babel-loader` in your dependencies as well because we list these as a peer dependencies:
```

Comment:
Why babel-core ?

Reply:
Damn, that was a mistake that carried over from an old version. It was fixed on line 38, but we missed it here.

Reply:
And on line 19. :|

Reply:
Yeah, I can make another PR.